Document upgrade process for 0.x to 1.x (#303)
vshn/k8up: docs/modules/ROOT/pages/explanations/missing-docs.adoc

= Missing Documentation

For the following keywords documentation is missing and will be added over time.

> This should give you an idea what is included, but currently invisible in the Operator.

Reviewer: ```suggestion
This should give you an idea what is included, but currently not (properly) documented.
```
